android/app/src/main/res/values/styles.xml

```diff
@@ -5,7 +5,11 @@
         <item name="android:navigationBarColor">@color/splashBackground</item>
         <item name="android:forceDarkAllowed">false</item>
     </style>
-    
+
+    <style name="ShareTheme" parent="AppTheme">
```

Comment:
How can I reproduce this change?

Reply:
When sharing a file from share extension. PR contains a GIF with what changes.